### PR TITLE
bug/7393-Liz-update-notification-screen-copy

### PR DESCRIPTION
**Pull request by liztownd**
## Description of Change
Updated the copy on the Notification settings screen.

## Screenshots/Video

Before/after (Android): 

&nbsp;&nbsp;

After on iOS (Sorry no phone outline, only the screen showed in the screen shot):

## Testing
Also updated unit tests and e2e tests.

- [x] Tested on iOS 
- [x] Tested on Android 

## Reviewer Validations
Copy should reflect what was requested in the [ticket #7393](https://github.com/department-of-veterans-affairs/va-mobile-app/issues/7393).

## PR Checklist

  **Reviewer:** Confirm the items below as you review

- [ ] PR is connected to issue(s)
- [ ] Tests are included to cover this change (when possible)
- [ ] No magic strings (All string unions follow the [Union -> Constant](https://github.com/department-of-veterans-affairs/va-mobile-app/blob/develop/VAMobile/src/constants/common.ts) type pattern)
- [ ] No secrets or API keys are checked in
- [ ] All imports are absolute (no relative imports)
- [ ] New functions and Redux work have proper TSDoc annotations

## For QA

[Run a build for this branch](https://github.com/department-of-veterans-affairs/va-mobile-app/actions/workflows/on_demand_build.yml)

### CODE REVIEW

File: VAMobile/src/translations/en/common.json [hunk 1]
```diff
@@ -1386,4 +1386,4 @@
   "whatsNew.bodyCopy.2.13": "You can now show proof of your Veteran status with the app.",
   "whatsNew.dismissMessage": "Dismiss this message",
   "whatsNew.title": "What's new"
-}
+}
```
Comment (Sparowhawk) [inline VAMobile/src/translations/en/common.json:1389]: need to add a new line it looks like

**Review by Sparowhawk — CHANGES_REQUESTED** [review]

File: VAMobile/src/translations/en/common.json (hunk 1 above)
Comment (liztownd) [reply VAMobile/src/translations/en/common.json:1389]: I did not actively delete the new line. I noticed it was missing, added it back in, hit save, and it is automatically deleted by prettier.

**Review by Sparowhawk — APPROVED** [review]

**Comment by TKDickson** [comment]
This isn't ready to test, see comments in 7393 ticket. Setting label back to "requested changes" and Liz can update when she's ready for PR review again.

**Comment by liztownd** [comment]
I left a comment over on the ticket as well but I'll drop a comment here as well. The 2 places that the text was not updated are coming directly from the API call to the VEText endpoint. Not sure if this is something FE needs to handle, or if it goes to BE to update, or if it's something from VA itself (via VEText) that will need to be adjusted....

**Review by DJUltraTom — APPROVED** [review]

**Pull request merged**